src/nupic/research/spatial_pooler.py

```diff
@@ -1505,9 +1506,15 @@ def _getColumnNeighborhood(self, centerColumn):
     @returns (1D numpy array of integers)
     The columns in the neighborhood.
     """
```

Comment:
The docstring should be updated to match the other one. `"Simply calls topology.wrappingNeighborhood or topology.neighborhood."`